src/control/destination/background/mod.rs

```diff
@@ -333,4 +347,33 @@ where
             }
         })
     }
+
+    fn tls_controller_ns(&self) -> Option<&str> {
+        self.namespaces.tls_controller.as_ref().map(String::as_ref)
+    }
+}
+
+// ===== impl DestinationCache =====
+
+impl<T> DestinationCache<T>
+where
+    T: HttpService<RequestBody = BoxBody, ResponseBody = RecvBody>,
+    T::Error: fmt::Debug,
+{
+    fn new() -> Self {
```

Comment:
tioli derive default; no need for this constructor, really?

Reply:
If I recall correctly, I originally tried to derive `Default` for this, but can't because `T` doesn't implement `Default`. It shouldn't be _necessary_ for it to, since the default HashMap should be empty, but I think that's still required to derive it.

Reply:
Huh, nevermind, I'm wrong. Looks like it can derive `Default`. Will change that, thanks!

Reply:
you could manually impl Default but it's probably not worth it. sgtm.

Reply:
Yeah, I thought this was better than adding another `impl` block + import just to get `Default`, since it isn't a member of anything that itself requires `Default`...